### DISCUSSION TITLE
No API and webUI access after latest update.

### DISCUSSION THREAD

**Discussion by Sighi-04** [solectrus/solectrus#3277] [Q&A]
Hello,
presumably after an update, I lost access to both the API and webUI. I am using Senec Home V3 Hybrid. Both just respond with 504 gateway timeout. Tried both http and https. Looks like it stopped working yesterday around 6AM. Is anyone experiencing the same? If I ever get the API back, I'm considering blocking internet access to the device at this point. Any feedback is appreciated, thanks.

**Comment by ledermann** [comment]
Apparently you mean the SENEC device and not SOLECTRUS. Or are you talking about the recently released version 0.15 of SOLECTRUS?

If I couldn't access my SENEC device via network, the first thing I would do is look at the hardware display. Maybe the device has lost its network connection. Maybe restarting the device or restarting the internet router can help.

**Reply by Sighi-04** [reply]
Yes, sorry if I didn't make that very clear. The device is online, I have also tried rebooting it. The port is still open, but no webUI or API are available, I just get 504 (which is coming from the internal nginx proxy on the senec device). Also tried moving the device to a different subnet to see if that changed anything with no luck. I am unable to verify which firmware is now running since it cannot be checked on the display itself. I was wondering if any user here is having the same issue.

**Reply by ledermann** [reply]
Apparently I can't help, my device works fine, and it has never thrown a 504 error. I'm curious to see if and how the problem can be solved.

**Comment by Zeppelin-PV** [comment]
... for better understanding
the device (SENEC) ist working ?
do you have accsess with the SENEC App ?
do you have accsess via online dashboard (https://mein-senec.de/endkunde/#/0/status) ?
did you get npu errors ?
what does your display show ? (upper line with ip adress)
![grafik](https://github.com/user-attachments/assets/812bec7c-1ebb-4f3d-85c6-141e1044d573)

**Reply by Zeppelin-PV** [reply]
on my router (FritzBox) there is only Port 80 shown
![grafik](https://github.com/user-attachments/assets/e10436a9-ab04-44a3-8aaf-851e90053c02)

**Reply by Sighi-04** [reply]
Yes, the device is working correctly. Access via online dashboard and mobile app works fine. No errors and the display is just like yours, except with my IP address on. Port 443 is open. I don't know why you still have port 80 tho, they turned it off in favour of HTTPS months ago at this point.

<img width="682" alt="Screenshot 2024-07-19 at 14 36 16" src="https://github.com/user-attachments/assets/2e8c73e8-301d-498c-a6fa-29bc254d4fd9">

<img width="1710" alt="Screenshot 2024-07-19 at 14 54 06" src="https://github.com/user-attachments/assets/88e688b2-cd7f-4e9e-a8eb-2aac64d6fbc8">

**Reply by gereons** [reply]
Just to rule out Firefox as the error's source, can you try running `curl -vik https://10.10.10.11` and post the output here?

**Reply by Sighi-04** [reply]
<img width="682" alt="Screenshot 2024-07-19 at 17 26 26" src="https://github.com/user-attachments/assets/364438ad-20b8-4114-9ed8-77f6fee32cf4">
Also tried to contact the api with postman, no luck. It seems either something on my board broke or an update removed access to API and local webGUI

**Reply by gereons** [reply]
Yeah, looks very much like a hardware issue and you'll need to get your system serviced. With any luck, only the flash card that stores all software internally has gone bad.

**Comment by Zeppelin-PV** [comment]
... yes, you are right. usually SENEC runs on https, but my npu-board was down, when they changed the board they did not set up for http to https.
for me, this is ok
on your side, i'm wondering about your ip adresses, usually home-net adresses running on 192.168.x.x
sorry, no idea what's wrong on your home network

**Reply by Sighi-04** [reply]
Nothing is wrong on my home network, that's how I set up my subnet, it has been this way for years now. This is more of a senec related issue, I was wondering if other people got the same update, but seems like I'm the only one at the moment.

**Comment by Sighi-04** [comment]
Hello,
I am closing the discussion as I just noticed that two days ago at exactly 19.30 the API was available again. So I think this is just a bug on my device and not a limitation of the new update.
